roles/ncn-common/files/scripts/common/create-ims-initrd.sh

```diff
@@ -42,4 +42,7 @@ rm -f /squashfs/*
 cp -pv /boot/vmlinuz-${KVER} /squashfs/${KVER}.kernel
 cp -pv /boot/initrd-${KVER} /squashfs/initrd.img.xz
 
+echo "Purging old kdumps initrd; kdump.service will generate a new one on first boot."
+rm -f /boot/initrd-*-kdump
```

Comment:
I went with the `*` but in reality this could be `initrd-${KVER}-kdump`.

I only chose the `*` because surtur has two kdump initrds right now, but we fixed the multiple kernel issue in the newer 1.3 images (and 1.4).. so I could depend on that and change this to `initrd-${KVER}-kdump`.